### DISCUSSION TITLE
How can I use llana over HTTPS instead of HTTP?

### DISCUSSION THREAD

**Discussion by mistaox** [juicyllama/llana#166] [General]
As the title states, how do I run llana over HTTPS?
I'm running llana on my own VPS and can't find a way to run it over HTTPS.
i have a self-signed certificate. Where do I put it?
The documentation makes no mention of this.

**Comment by andyslack** [comment]
Hi @mistaox 

Based on your comment on YouTube, you're running on a Windows Server.

There are a few suggestions which might be helpful about how to run HTTPS with docker: https://stackoverflow.com/questions/50810165/run-docker-service-on-https

I am not familiar with running Docker on Windows, if you want to get Llana up and running quickly, I would recommend following the guide here: https://llana.io/hosting

Hopefully, somebody else with more experience with Docker and Windows Server can comment soon. 

If you find a solution, please let us know.

**Reply by mistaox** [reply]
Thanks for the reply. Great project!

I'm very much interested in self-hosting  llana, but I'm not actually using docker.
I'm running it with just node.js along with the nextjs dependency.
it runs fine this way.  
I was hoping to use it in conjunction with a bolt.new project but the web container will not allow traffic over HTTP.

**Reply by andyslack** [reply]
I am delighted to hear you are seeing value in the project.
